diffcp/cone_program.py

```diff
@@ -251,10 +251,17 @@ def solve_and_derivative_internal(A, b, c, cone_dict, warm_start=None,
         data["s"] = warm_start[2]
 
     kwargs.setdefault("verbose", False)
+    kwargs.setdefault("acceleration_lookback", 0)
+    kwargs.setdefault("max_iters", 10000)
     result = scs.solve(data, cone_dict, **kwargs)
 
-    # check status
     status = result["info"]["status"]
+    if status == "Solved/Inaccurate" and "acceleration_lookback" not in kwargs:
```

Comment:
Doesn't `setdefault` above always add `acceleration_lookback` to `kwargs`? I'm not sure the code in this block does what you want.